### PR TITLE
Add Changelog for October 2023

### PR DESCRIPTION
**Pull request by fill-the-fill**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Updating documentation or Bugfix

Screenshot 2023-11-03 at 11 20 11

### CODE REVIEW

**Review by os11k — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1458,4 +1458,37 @@ All of the merged PRs in September
...
+
+#### Contributors: 9
+
+- rphair ([@rphair](https://github.com/rphair))
+- MladenLm ([@MladenLm](https://github.com/MladenLm))
+- fill-the-fill ([@fill-the-fill](https://github.com/fill-the-fill))
+- weqanhet ([@weqanhet](https://github.com/weqanhet))
+- mcmoodoo ([@mcmoodoo](https://github.com/mcmoodoo))
+- katomm ([@katomm](https://github.com/katomm))
+- dependabot ([@dependabot](https://github.com/dependabot))
```
Comment (rphair) [inline CHANGELOG.md:1492]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1458,4 +1458,37 @@ All of the merged PRs in September
...
+- Remove Developer Ecosystem 2023 Survey by ([@katomm](https://github.com/katomm)) in [#1165](https://github.com/cardano-foundation/developer-portal/pull/1165)
+- Update 2023-10-16-october.md by ([@katomm](https://github.com/katomm)) in [#1166](https://github.com/cardano-foundation/developer-portal/pull/1166)
+- Bump postcss from 8.4.23 to 8.4.31 by ([@dependabot](https://github.com/dependabot)) in [#1160](https://github.com/cardano-foundation/developer-portal/pull/1160)
+- Add DROPSPOT to Showcase by ([@fill-the-fill](https://github.com/fill-the-fill)) in [#1158](https://github.com/cardano-foundation/developer-portal/pull/1158)
+- Bump babel/traverse from 7.21.5 to 7.23.2 by ([@dependabot](https://github.com/dependabot)) in [#1167](https://github.com/cardano-foundation/developer-portal/pull/1167)
+- Aiken get started: Fix broken examples link by ([@OlofBlomqvist](https://github.com/OlofBlomqvist)) in [#1168](https://github.com/cardano-foundation/developer-portal/pull/1168)
+- Cardano Relay Configuration: fix & clarify instructions by ([@sanskys](https://github.com/sanskys)) in [#1162](https://github.com/cardano-foundation/developer-portal/pull/1162)
+
+#### Contributors: 9
```
Comment (rphair) [inline CHANGELOG.md:1484]:
```suggestion
#### Contributors: 8
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1458,4 +1458,37 @@ All of the merged PRs in September
...
+- Update Docusaurus to 2.4.3 by ([@fill-the-fill](https://github.com/fill-the-fill)) in [#1154](https://github.com/cardano-foundation/developer-portal/pull/1154)
+- Add DexHunter to Showcase by ([@fill-the-fill](https://github.com/fill-the-fill)) in [#1159](https://github.com/cardano-foundation/developer-portal/pull/1159)
+- Add Lenfi Developer Blog  by ([@weqanhet](https://github.com/weqanhet)) in [#1163](https://github.com/cardano-foundation/developer-portal/pull/1163)
+- Smart Contracts overview: bit of language cleanup by ([@mcmoodoo](https://github.com/mcmoodoo)) in [#1164](https://github.com/cardano-foundation/developer-portal/pull/1164)
+- Remove Developer Ecosystem 2023 Survey by ([@katomm](https://github.com/katomm)) in [#1165](https://github.com/cardano-foundation/developer-portal/pull/1165)
+- Update 2023-10-16-october.md by ([@katomm](https://github.com/katomm)) in [#1166](https://github.com/cardano-foundation/developer-portal/pull/1166)
+- Bump postcss from 8.4.23 to 8.4.31 by ([@dependabot](https://github.com/dependabot)) in [#1160](https://github.com/cardano-foundation/developer-portal/pull/1160)
```
Comment (rphair) [inline CHANGELOG.md:1478]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1458,4 +1458,37 @@ All of the merged PRs in September
...
+- Add DROPSPOT to Showcase by ([@fill-the-fill](https://github.com/fill-the-fill)) in [#1158](https://github.com/cardano-foundation/developer-portal/pull/1158)
+- Bump babel/traverse from 7.21.5 to 7.23.2 by ([@dependabot](https://github.com/dependabot)) in [#1167](https://github.com/cardano-foundation/developer-portal/pull/1167)
```
Comment (rphair) [inline CHANGELOG.md:1480]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1458,4 +1458,37 @@ All of the merged PRs in September
...
 - katomm ([@katomm](https://github.com/katomm))
-- gufmar ([@gufmar](https://github.com/gufmar))
\ No newline at end of file
+- gufmar ([@gufmar](https://github.com/gufmar))
+
+## 2023.43.0 (2023-10-31)
+
+All of the merged PRs in October
+
+#### PRs: 14
```
Comment (rphair) [inline CHANGELOG.md:1467]:
```suggestion
#### PRs: 12
```

**Review by rphair — COMMENTED** [review]
There was an argument for removing @dependabot which came up last month & I can't recall the consensus, but I think a strong case was made for removing him from lists of contributors (sorry old buddy 🤖)  cc @katomm

**Review by rphair — APPROVED** [review]

**Review by rdlrt — APPROVED** [review]

**Pull request merged**